include/deal.II/base/partitioner.templates.h

```diff
@@ -281,6 +283,31 @@ namespace Utilities
 
 
 
+    namespace
```

Comment:
Lately, we try to avoid unnamed namespaces in header files. I know that you are not defining any variables here, but would you name it `internal` or the like for consistency?

Reply:
I thought we only apply that rule to header files where an anonymous namespace might be confusing/useless? This one is a `.templates.h` file that will not get regularly included. But I'm fine changing it anyway. Do you remember if we said something about our preferred scheme in those implementation headers?

Reply:
No, I don't remember. I just extrapolated from the `*.h` header files since these files in the end might be used the same way. 

On the other hand, there are examples of unnamed namespaces in the other `*.templates.h` as well. So I don't feel too strong about this.

Reply:
I think that generally, anonymous namespaces in header files are asking for trouble. That's true for any file that may be `#include`d into more than one place, and so I guess that also includes `.template.h` files (though I don't think it's an urgent issue there).